src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/AutoInsert/OnAutoInsertEndpointTest.cs

```diff
@@ -40,7 +40,11 @@ public async Task Handle_SingleProvider_InvokesProvider()
             {
                 TextDocument = new TextDocumentIdentifier(uri),
                 Character = ">",
-                Options = new FormattingOptions(),
+                Options = new FormattingOptions
+                {
+                    TabSize = 4,
+                    InsertSpaces = true
+                },
```

Comment:
Looks like its only luck this didn't come up before, but `new FormattingOptions().TabSize` will throw a `KeyNotFoundException`. Will follow up with LSP platform.

Reply:
Looks like this is already fixed in OmniSharp: https://github.com/OmniSharp/csharp-language-server-protocol/blob/master/src/Protocol/Models/FormattingOptions.cs#L21